### ISSUE TITLE
Enhance detail and reproducibility of instructions in readme

### ISSUE THREAD

**Issue by bnord** [deepskies/DeepUQ#201]
I think the readme is a bit hard to follow.

1. I suggest making every line copy-and-paste-able once the code has been installed via pip. 
2. 3. It's probably easier for the user to just put a name here, like `deepuqstudy` or something like that.
```
python3.10 -m venv name_of_your_virtual_env
source name_of_your_virtual_env/bin/activate
```
3. Do people still use virtualenv? I thought maybe conda replaced that?
4. The organization of the readme is a bit tough to follow because the sections aren't very clear. I suggest using more levels and more white space.

**Comment by beckynevin** [comment]
@bnord as a reminder, the format of the README as well as the venv stuff is from DeepTemplate-Science. If you'd like me to follow another standard for this, let me know, but just be aware that that's where this is from ([https://github.com/deepskies/DeepTemplate-Science](https://github.com/deepskies/DeepTemplate-Science)).

**Comment by beckynevin** [comment]
Addressed in PR #204
